task/upload-sbom-to-trustification/0.1/upload-sbom-to-trustification.yaml

```diff
@@ -249,3 +264,13 @@ spec:
           --data "@$supported_version_of_sbom" \
           "$bombastic_api_url/api/v1/sbom?id=$sbom_id"
       done
+
+  - name: report
+    image: quay.io/konflux-ci/appstudio-utils:48c311af02858e2422d6229600e9959e496ddef1@sha256:91ddd999271f65d8ec8487b10f3dd378f81aa894e11b9af4d10639fd52bba7e8
```

Comment:
In the meantime, this got updated to `quay.io/konflux-ci/appstudio-utils:8f9f933d7b0b57e37b96fd34698c92c785cfeadc@sha256:924eb1680b6cda674e902579135a06b2c6683c3cc1082bbdc159a4ce5ea9f4df`